server2/Arcadia.Assistant/Arcadia.Assistant.Sharepoint/SharepointItemSynchronization.cs

```diff
@@ -93,7 +126,7 @@ private async Task<StorageItem> GetSharepointItemForCalendarEvent(IExternalStora
             return existingItems.SingleOrDefault();
         }
 
-        private async Task UpsertStorageItem(string eventId, string calendar, DatesPeriod datesPeriod, StorageItem upsertItem, IExternalStorage externalStorage, CancellationToken cancellationToken)
+        private async Task UpsertStorageItem(string eventId, string calendar, EmployeeId employeeId, DatesPeriod datesPeriod, StorageItem upsertItem, IExternalStorage externalStorage, CancellationToken cancellationToken)
```

Comment:
neither datesPeriod nor employeeId is used

Reply:
Fixed.